docs/appendices/release-notes/5.4.1.rst

```diff
@@ -147,6 +147,11 @@ Fixes
   ``pg_catalog.pg_views`` and ``information_schema.views`` tables to denote
   that a ``VIEW``'s query is erroneous.
 
+- Fixed an issue that caused query on un-indexed columns to return empty
+  results. The expected behaviour was to have an exception thrown but as part
+  of this fix, un-indexed columns are now queryable. Also note that this fix
```

Comment:
No need for the last sentence, but we can adjust before merging everything to master.